src/pineko/fonll.py

```diff
@@ -235,7 +239,7 @@ def collect_updates(fonll_fns, damp):
     """Produce the different theory cards according to which FONLL is asked for."""
     updates = []
     is_mixed = fonll_fns in MIXED_ORDER_FNS
-    is_damped = damp != 0
+    is_damped = damp == 1
```

Comment:
```suggestion
    is_damped = damp != -1
```

Reply:
It doesn't look correct to me, since we agreed that `0` should be *damped* (in the sense of the `theta` function, and thus the extra sub-tables).

> [cheezburger.com/6060857344/it-hurt-itself-in-its-confusion](https://cheezburger.com/6060857344/it-hurt-itself-in-its-confusion)

I perfectly agree with @giacomomagni (and if it's a benchmark problem, most likely the benchmarks will be outdated)

Reply:
I suspect the variable should be called `is_non_trivially_damped`, but better @andreab1997 should comment

Reply:
@giacomomagni are you doing that here? Otherwise I'll do it as soon as I can